src/codeu/chat/common/ServerVersion.java

```diff
@@ -0,0 +1,31 @@
+package codeu.chat.common;
```

Comment:
I think all of this file is pretty much alongside ServerInfo.java
No need to split it up